src/qt/clientmodel.cpp

```diff
@@ -226,12 +240,19 @@ static void NotifyAlertChanged(ClientModel *clientmodel, const uint256 &hash, Ch
                               Q_ARG(int, status));
 }
 
+static void BannedListChanged(ClientModel *clientmodel)
+{
+    qDebug() << "BannedListChanged";
```

Comment:
You could also use something like:
``qDebug() << QString("%1: Requesting update for peer banlist").arg(__func__);`` I intend to creat a pull to make use of that format so we have the debug logging include always the correct function name :).